### PR TITLE
Sjg misc decomp

### PR DESCRIPTION
**Pull request by stephenJohnGiardina**

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀🚀 This PR matches 3 functions (+0.01%) on `us`.

File: src/world/area_kzn/kzn_02/C5AA50.c [hunk 1]
```diff
@@ -1,9 +1,45 @@
...
+ApiStatus func_80240310_C5AA50(Evt* script, s32 isInitialCall) {
+    Bytecode* args = script->ptrReadPos;
+    f32 sp20;
+    f32 sp24;
+    f32 sp28;
+    f32 sp2C;
+    f32 sp30;
+    f32 sp34;
+
+    get_model_center_and_size(evt_get_variable(script, *args++), &sp20, &sp24, &sp28, &sp2C, &sp30, &sp34);
```
Comment (pixel-stuck) [inline src/world/area_kzn/kzn_02/C5AA50.c:12]: Forgot to mention this, but check this function's prototype, if the args are named (e.g. something more descriptive than arg0, etc.) then you can name your vars the same here (e.g. sp20 and co would have the same names as the arguments)

File: src/world/area_kpa/kpa_04/A4BF60.c [hunk 2]
```diff
@@ -5,4 +5,30 @@ static char* N(exit_str_1) = "kpa_81";
 
 #include "world/common/UnkFunc11.inc.c"
 
-INCLUDE_ASM(s32, "world/area_kpa/kpa_04/A4BF60", func_802400BC_A4C01C);
+ApiStatus func_802400BC_A4C01C(Evt *script, s32 isInitialCall) {
```
Comment (pixel-stuck) [inline src/world/area_kpa/kpa_04/A4BF60.c:8]: Star on the left rather than the right

**Comment by pixel-stuck** [comment]
Looks like you've got a merge conflict now, for some reason; you'll need to resolve that before we can merge this

File: src/world/area_kzn/kzn_02/C5AA50.c [hunk 3]
```diff
@@ -1,9 +1,45 @@
...
+    PlayerStatus* playerStatus = &gPlayerStatus;
+    CollisionStatus* collisionStatus= &gCollisionStatus;
+    Npc* partner;
 
-INCLUDE_ASM(s32, "world/area_kzn/kzn_02/C5AA50", func_802403DC_C5AB1C);
+    if ((collisionStatus->currentFloor == temp_s1) || ((collisionStatus->lastTouchedFloor == temp_s1)) || (collisionStatus->currentFloor == temp_s0_3) || (collisionStatus->lastTouchedFloor == temp_s0_3)) {
+        playerStatus->extraVelocity.x = temp_s2;
+    }
+    if (gPlayerData.currentPartner != 0){
+        partner = get_npc_unsafe(-4);
```
Comment (nanaian) [inline src/world/area_kzn/kzn_02/C5AA50.c:39]:
```suggestion
        partner = get_npc_unsafe(NPC_PARTNER);
```

File: src/world/area_kzn/kzn_02/C5AA50.c [hunk 4]
```diff
@@ -1,9 +1,45 @@
...
+    s32 temp_s0_3 = evt_get_variable(script, *args++);
+    PlayerStatus* playerStatus = &gPlayerStatus;
+    CollisionStatus* collisionStatus= &gCollisionStatus;
+    Npc* partner;
 
-INCLUDE_ASM(s32, "world/area_kzn/kzn_02/C5AA50", func_802403DC_C5AB1C);
+    if ((collisionStatus->currentFloor == temp_s1) || ((collisionStatus->lastTouchedFloor == temp_s1)) || (collisionStatus->currentFloor == temp_s0_3) || (collisionStatus->lastTouchedFloor == temp_s0_3)) {
+        playerStatus->extraVelocity.x = temp_s2;
+    }
+    if (gPlayerData.currentPartner != 0){
```
Comment (nanaian) [inline src/world/area_kzn/kzn_02/C5AA50.c:38]:
```suggestion
    if (gPlayerData.currentPartner != PARTNER_NONE) {
```

**Review by nanaian — CHANGES_REQUESTED** [review]

**Pull request merged**